colossalai/booster/plugin/hybrid_parallel_plugin.py

```diff
@@ -356,6 +391,7 @@ def configure(
         dataloader: Optional[DataLoader] = None,
         lr_scheduler: Optional[LRScheduler] = None,
     ) -> Tuple[Module, OptimizerWrapper, Callable, DataLoader, LRScheduler]:
+        param_info = get_param_info(optimizer)
```

Comment:
Can you get param info in `__init__()` of optimizer wrapper?

Reply:
The model has been sharded when optimizer is wrapped. To keep the information of parameter shape, the param info should be obtained before model has been sharded.